code/modules/mob/transform_procs.dm

```diff
@@ -281,6 +281,27 @@
 	qdel(src)
 	return new_corgi
 
+/mob/living/carbon/human/proc/crabize()
```

Comment:
hate to be this guy but you've got some undocumented procs and single letter vars here. be a hero? save the code